### ISSUE TITLE
[BUG] NEXT buttons used on Tabbed navigation pages not working. 

### ISSUE THREAD

**Issue by dkevey** [unimelb/design-system-issues#63]
## Summary
Using next button on tabbed pages not working correctly. See [FAQ](https://resources.web.unimelb.edu.au/controlled-environment/templates/home/home-page-with-tabbed-navigation#faq) on how to implement in CMS.

## Expected behaviour
Example page: https://melbourne-cshe.unimelb.edu.au/programs/engagement/engaged-university-esymposium

On About tab, when clicking on  "Next: Forum Hosts" button expect to go to "Get Involved" tab and have it load like this: 
![Screen Shot 2020-04-28 at 8 24 26 am](https://user-images.githubusercontent.com/1305804/80426790-c6c9cd80-8929-11ea-8e2f-de3073156653.png)

![Screen Shot 2020-04-28 at 8 25 40 am](https://user-images.githubusercontent.com/1305804/80427263-b49c5f00-892a-11ea-9864-cdb8f71f5ef6.png)

## Current behaviour
But instead the required element is not targeted correctly
![Screen Shot 2020-04-28 at 8 23 50 am](https://user-images.githubusercontent.com/1305804/80426797-ccbfae80-8929-11ea-9c7e-b2c45909b753.png)

![Screen Shot 2020-04-28 at 8 26 23 am](https://user-images.githubusercontent.com/1305804/80427271-b82fe600-892a-11ea-9aaf-32c2c820875b.png)

**Comment by woowoowoo** [comment]
The same issue was reported on TASK0282732, referring to the opposite scrolling effect - in that a click on the tab should bring the tab bar up to the top of the viewport.

**Comment by tomstringer** [comment]
@woowoowoo I actually think that functionality was deliberately removed, is that right @jaredh-um ?

**Comment by woowoowoo** [comment]
Hmm - I actually think it is not good behaviour, and needs to be addressed. The original behaviour of the design system seemed correct to me. It's not so much about using extra buttons to traverse the tabs, as it is about scrolling to the tab bar, and bringing content into the viewport correctly.

**Comment by jaredh-um** [comment]
The issue where it was removed is here: https://github.com/unimelb/Gen3/issues/33

I didn't realise this behaviour was attached to buttons as well; I originally noticed it when clicking on the tabs themselves.

I don't think it should be reinstated for the tab links themselves; shifting the viewport when the content is already in view seems a bit overbearing to me. However if it's on a button of unknown location, then this would make more sense.

**Comment by woowoowoo** [comment]
Even a click on the tab - if that tab bar close to the bottom of the viewport - you don't really see the content update within the viewport - that's the problem that was reported today. Edge case, but sort of fair.

**Comment by jaredh-um** [comment]
Isn't it reasonable to expect that clicking on a tab changes the content underneath? And if you've chosen to click it when it's at the bottom of the page that you're probably going to have to scroll to see what changed? That's pretty standard tab behaviour.

**Comment by woowoowoo** [comment]
Yes, but isn't it equally reasonable to assume that a user clicking on a tab wants to move that content into the viewport? One interaction instead of two.

**Comment by dkevey** [comment]
@jaredh-um this issue has come up a few times and agree with Andrew that the functionality should be resinated as it was prior to the upgrade. 

Historically this functionality was enabled editors to display short courses in their sites as Bespoke didn't cater for these. An example of how a course would work in MCE is here: https://resources.web.unimelb.edu.au/controlled-environment/templates/special/course-filtered-list/music/certificate-in-excellence Because tabbed navigation was new (at the time) the idea was what when users scrolled to the bottom of the page the NEXT button was a quick and easy way to keep them moving through the content to the next tab. So the way the content is set up for this program, it's still relevant https://melbourne-cshe.unimelb.edu.au/programs/engagement/engaged-university-esymposium

Ideally if the plan is to remove tabbed navigation entirely then we no longer need the NEXT buttons, but we would need wider comms around this change and offer some alternative. 

At the moment this is a bug which we should fix when we get the chance please.

**Comment by jaredh-um** [comment]
I agree that it should be reinstated, however this javascript is a kludge to cover up some rather poor design decisions (though perhaps appropriate in that era) that naturally result in poor usability.

I don't think that most of the interactions around this element result in anything a user would reasonably expect in 2020 as a result. It is amongst the top items I want to see replaced if we get a new design system online.

@gepifanio are you able to look into reverting [this issue](https://github.com/unimelb/Gen3/issues/33)? We would like to reinstate the original behaviour.

**Comment by tomstringer** [comment]
@gepifanio This has been raised again. Are we able to get this particular change reverted? See @jaredh-um's comment above.
